backend-api/README.md

```diff
@@ -36,14 +36,18 @@ npm run test
 1. Activate AWS credentials
 2. Deploy your stack
 3. Generate a `.env` file for your deployed stack
-
 ```bash
 # From /backend-api
 sh generate_env_file.sh <stack_name>
 ```
 
-4. Run tests
+4. Add the following environment variables to the `.env` file:
```

Comment:
Please may we use the generate_env_file.sh for this to make this experience as plug and play as possible :)

Reply:
Done